src/rules/importBlacklistRule.ts

```diff
@@ -237,7 +237,9 @@ function walk(ctx: Lint.WalkContext<Options>) {
                               toExportName,
                           )
                         : []),
-                    ...(exportClause !== undefined ? exportClause.elements.map(toExportName) : []),
+                    ...(exportClause && ts.isNamedExports(exportClause)
```

Comment:
> `ts.is*`

Quick tip for the build failures: because TSLint supports older versions of TypeScript (we're not as fortunate as typescript-eslint to have a less generous backwards compatibility story), you'll need to use `tsutils.is*` equivalents.

```suggestion
                    ...(exportClause && tsutils.isNamedExports(exportClause)
```

For the functions that also aren't in older, supported versions of `tsutils`, you'll need something like:

https://github.com/palantir/tslint/blob/079622400791b70eb9f132637bfe364e38a9058d/src/language/utils.ts#L581

Reply:
Thanks for the pointer! I think I fixed this up, and added a new util for `isPrivateIdentifier`.